community/gnome-desktop/APKBUILD

```diff
@@ -1,39 +1,41 @@
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=gnome-desktop
-pkgver=3.30.1
+pkgver=3.32.1
 pkgrel=0
 pkgdesc="gnome desktop core libraries"
-url="https://www.gnome.org/"
+url="https://gitlab.gnome.org/GNOME/gnome-desktop"
 arch="all"
-license="GPL"
-depends_dev="gvfs-dev libgudev-dev eudev-dev iso-codes-dev
-	gobject-introspection-dev"
-makedepends="$depends_dev xkeyboard-config itstool libxml2-utils gtk+3.0-dev libseccomp-dev
-	gsettings-desktop-schemas-dev gtk-doc py-six"
+license="GPL-2.0-or-later, LGPL-2.1-or-later"
+depends_dev="eudev-dev iso-codes-dev libseccomp-dev libx11-dev xkeyboard-config
+	gobject-introspection-dev gsettings-desktop-schemas-dev gtk+3.0-dev
+	"
+makedepends="$depends_dev itstool libxml2-utils gtk-doc py-six meson py3-setuptools
+	glib-dev"
+depends="bubblewrap"
+options="!check" # Can't find its own GSettings schemas
```

Comment:
This should be reported upstream if it hasn't been already

Reply:
https://gitlab.gnome.org/GNOME/gnome-desktop/issues/114